### PR TITLE
Add template to the widgets

### PR DESCRIPTION
**Pull request by bjorvack**
Make it possible to have different templates for each widget

### CODE REVIEW

File: src/Backend/Modules/Partners/Domain/Widget/WidgetType.php [hunk 1]
```diff
@@ -52,4 +70,30 @@ public function configureOptions(OptionsResolver $resolver)
...
+     * @return array
+     */
+    private function getPossibleTemplates()
+    {
+        $templates = array();
+        $finder = new Finder();
+        $finder->name('*.html.twig');
+        $finder->in(FRONTEND_MODULES_PATH . '/Partners/Layout/Widgets');
+        // if there is a custom theme we should include the templates there also
+        $theme = Model::get('fork.settings')->get('Core', 'theme', 'core');
```
Comment (carakas) [inline src/Backend/Modules/Partners/Domain/Widget/WidgetType.php]: in theory you should inject fork.settings via the constructor and create a service of the form but I'm willing to let this one slip, dunno bout @StijnVrolijk

**Review by carakas — APPROVED** [review]

File: src/Backend/Modules/Partners/Actions/Edit.php [hunk 2]
```diff
@@ -22,7 +22,9 @@ public function execute()
         }
 
         $editWidget = new UpdateWidget($widget);
-        $form = $this->createForm(WidgetType::class, $editWidget);
+
+        $theme = $this->get('fork.settings')->get('Core', 'theme', 'core');
+        $form = $this->createForm(new WidgetType($theme), $editWidget);
```
Comment (carakas) [inline src/Backend/Modules/Partners/Actions/Edit.php]: this is deprecated so we can't accept it
you should pass the fork.settings service as a constructor parameter and then define it in a services.yml or something
that way it is automatically handled for you and you can keep using WidgetType::class like you should

**Review by carakas — CHANGES_REQUESTED** [review]

File: src/Backend/Modules/Partners/Actions/Edit.php (hunk 2 above)
Comment (bjorvack) [reply src/Backend/Modules/Partners/Actions/Edit.php]: fixed 3429477

File: src/Backend/Modules/Partners/Actions/Edit.php [hunk 3]
```diff
@@ -22,6 +22,7 @@ public function execute()
         }
 
         $editWidget = new UpdateWidget($widget);
+
```
Comment (carakas) [inline src/Backend/Modules/Partners/Actions/Edit.php]: why did you add an enter here?

**Review by carakas — CHANGES_REQUESTED** [review]

File: src/Backend/Modules/Partners/Actions/Edit.php (hunk 3 above)
Comment (bjorvack) [reply src/Backend/Modules/Partners/Actions/Edit.php]: fixed 	76a5736

**Review by carakas — APPROVED** [review]

**Comment by bjorvack** [comment]
@StijnVrolijk nog remarks?

**Pull request merged**